app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,16 @@
+<h1><%= @task.title %></h1>
+<article>
+  <h4>Description: </h4>
+  <p>
+    <%= @task.description %>
+  </p>
+  <h4>Status: </h4>
+  <p>
+    <%= @status %>
+  </p>
+</article>
+<div>
```

Comment:
@brianaeng Do you show a button to mark it as complete even if it is complete?  Do you need to?